crates/sui-adapter/src/programmable_transactions/types.rs

```diff
@@ -239,6 +236,10 @@ impl ObjectValue {
             ObjectContents::Coin(coin) => buf.extend(coin.to_bcs_bytes()),
         }
     }
+
+    pub fn into_type(self) -> MoveObjectType {
```

Comment:
I'm confused why this function exists, the field is public?

Reply:
Fair point, will eliminate.